app/lib/controlled_work_mappings/cw2_welsh.yml

```diff
@@ -18,6 +18,12 @@
       yes_value: "Yyd"
       no_value: "Na"
 
+    - name: Under 18
+      type: boolean_radio
+      source: under_eighteen
+      yes_value: Yyd
```

Comment:
i know this is just a draft PR, but thought I would mention it: the yes value looks llike a spelling error, it would usualy be 'ydy' although I dont put it past the form have the spelling error and the value therefore being correct.

Reply:
I've added a note to explain this.